src/Sarif.Viewer.VisualStudio.UnitTests/OutputWindowTracerListenerTests.cs

```diff
@@ -0,0 +1,55 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+using System.Collections.Generic;
+using System.Collections.ObjectModel;
+using System.Diagnostics;
+using System.Text;
+
+using FluentAssertions;
+
+using Microsoft.CodeAnalysis.Sarif;
+using Microsoft.VisualStudio.Shell.Interop;
+
+using Moq;
+
+using Xunit;
+
+namespace Microsoft.Sarif.Viewer.VisualStudio.UnitTests
+{
+    public class OutputWindowTracerListenerTests : SarifViewerPackageUnitTests
+    {
+        [Fact]
+        [System.Diagnostics.CodeAnalysis.SuppressMessage("Usage", "VSTHRD010:Invoke single-threaded types on Main thread", Justification = "For unit tests")]
+        public void OutputWindowTracerListener_WriteTests()
+        {
+            // arrange
+            string currentOutputString = null;
+
+            var mockPane = new Mock<IVsOutputWindowPane>();
+            IVsOutputWindowPane pane = mockPane.Object;
+            mockPane
+                .Setup(p => p.OutputStringThreadSafe(It.IsAny<string>()))
+                .Callback((string outputString) => currentOutputString = outputString);
+
+            var mockOutputWindow = new Mock<IVsOutputWindow>();
+            mockOutputWindow.Setup(o => o.CreatePane(ref It.Ref<Guid>.IsAny, It.IsAny<string>(), 1, 1)).Returns(0);
+            mockOutputWindow.Setup(o => o.GetPane(ref It.Ref<Guid>.IsAny, out pane));
+
+            // act
+            var outputWindowTraceListener = new OutputWindowTracerListener(mockOutputWindow.Object, "TestPane");
+
+            Trace.Write("Test log");
```

Comment:
>"Test log" [](http://example.com/codeflow?start=24&length=10)

Any room for a string variable to cut down on repetition?

Reply:
updated using a string variable